vspace/src/main/java/edu/asu/diging/vspace/core/factory/ISpaceFactory.java

```diff
@@ -1,10 +1,13 @@
 package edu.asu.diging.vspace.core.factory;
 
 import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.web.staff.forms.LocalizedTextForm;
 import edu.asu.diging.vspace.web.staff.forms.SpaceForm;
 
 public interface ISpaceFactory {
-
-    ISpace createSpace(SpaceForm form);
+    
+    public ISpace createSpace(SpaceForm form);
+    
+    public void addSpaceDescription(ISpace space, LocalizedTextForm description);
```

Comment:
we don't put `public` in front of interface methods since that's implicitly given.